### ISSUE TITLE
[Programmatic Access - Make code Arcade- Project-editor]: Incorrect heading structure is defined on the page.

### ISSUE THREAD

**Issue by BURRAshiva01** [microsoft/pxt-microbit#6118]
### User Impact:
Visually impaired people who rely on screen readers will be impacted while navigating the page in heading mode if the heading structure of the page is not defined in a logical sequence i.e., h1>h2>h3>h4>h5>h6.
 
**Note:** User credentials should Not be included in the bug. 

### Repro Steps:

1. Launch application URL: https://makecode.microbit.org/#editor
2. Now 'editor' page will open and navigate to the 'Git hub' control and select it.
3. Verify whether heading hierarchy is followed or not?

### Actual Result:
heading hierarchy is not followed on the page.

### Expected Result:
'burrashiva01/ff', 'History' and 'Release zone' text should define in h2 heading level.  

### MAS Reference:
[MAS 1.3.1 – Info and Relationships](https://liquid.microsoft.com/Web/Object/Read/ms.accessibility/Requirements/01.03.01#Zmore) 

### Reference Links: 

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier. 
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [Link](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”. 
- Please reach out to [C&AI Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fteams.microsoft.com%2fl%2fteam%2f19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype%2fconversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=04%7c01%7cv-vaisgupta%40microsoft.com%7c280d623b6a4d41db15ec08d9be06f891%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637749759434143108%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000&sdata=dka0fwBvuI9r4lnXxS4jvU6siHlCitqoSaY9%2Br3SeVc%3D&reserved=0) for any process related queries.

<img width="987" alt="Image" src="https://github.com/user-attachments/assets/0bfbb3cf-7ce1-420d-94af-c9e6dcc3d278" />

**Comment by Yamini234u** [comment]
As checked issue on https://makecode.microbit.org/#editor. The issue is still repro. Please find attachment for the reference

<img width="958" alt="Image" src="https://github.com/user-attachments/assets/904a693d-7ee8-4cc8-a93f-d6f770f934f0" />

**Comment by srietkerk** [comment]
This is fixed in beta. Please go to https://makecode.microbit.org/beta#editor to see the fix.
